src/components/accordion/CdrAccordion.vue

```diff
@@ -55,7 +54,8 @@ const props = defineProps({
   },
 });
 const emit = defineEmits(['accordion-toggle']);
-const unwrap = inject('unwrap');
+const unwrap = inject('unwrap', false);
+const isGrouped = inject('unwrap', false) ? 'li' : 'div';
```

Comment:
Added these default values to the injections because Vue was complaining when CdrAccordion wasn't slotted in to a group.